src/Presentation/RestApi/Service/RestApiTaskExecutor.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Presentation\RestApi\Service;
```

Comment:
Please, avoid using the `Service` folder. Use behaviour-specific naming instead. `Executor\Task` in this case.

Reply:
Fixed